CONCEPTS.md

```diff
@@ -37,6 +37,11 @@ thread.
 The gRPC programming surface in most languages comes in both synchronous and
 asynchronous flavors.
 
+### ABI compatibility
```

Comment:
this definitely doesn't belong here.

I think you can add a note to BUILDING.md (if you find no good place for it, feel free to append a section at the end of it).

Also, I think think it would be better to put this more in context. 1.) The SONAME only affects our cmake builds and 2.) the remark you wrote only describes the relationship between ABI compatibility and the SONAME.

But somehow as-is, this section sounds as if you're making general claims about ABI compatibility of gRPC and gRPC++ and that's not the intent. Please rephrase to make it clear what's the scope of the information this provides (e.g. we could call it "A note on SONAME and its ABI compatibility implications in the cmake build".

Reply:
Yep, moved as suggested and changed the title. I used the phasing almost as in https://github.com/grpc/grpc/pull/25617#issuecomment-799757679, whereby OFC it's only about this specific situation. As AJ told previously to have an intention writing up something with more details, the current version should be just to document the change as requested.

Thanks.

Reply:
Thank for updating the docs.